src/modules/signup/modal-additional-info/ModalAdditionalInfo.vue

```diff
@@ -84,7 +84,7 @@
             class="w-full h-3.5"
             :isError="!isValidEmail"
           />
-          <p v-if="!isValidEmail" class="text-sm text-red-5 mt-0.25 ml-1.5">
+          <p v-show="!isValidEmail" class="text-sm text-red-5 mt-0.25 ml-1.5">

```

Comment:
in `src\modules\signup\modal-additional-info\ModalAdditionalInfo.vue` we use v-if. Please select only one of both.

Reply:
Change to v-show better because user can type then remove, then type. It would be a waste to mount component multiple time.

Reply:
Then can you also change it additional modal info?